audiomentations/augmentations/air_absorption.py

```diff
@@ -0,0 +1,172 @@
+            x * 1e-3 for x in [0.1, 0.2, 0.5, 0.7, 1.4, 4.4, 15.8, 58.0, 214.9]
+        ],
+        "20C_30-50%": [
+            x * 1e-3 for x in [0.1, 0.3, 0.6, 1.0, 1.9, 5.8, 20.3, 72.3, 259.9]
+        ],
+        "20C_50-70%": [
+            x * 1e-3 for x in [0.1, 0.3, 0.6, 1.0, 1.7, 4.1, 13.5, 44.4, 148.7]
+        ],
+        "20C_70-90%": [
+            x * 1e-3 for x in [0.1, 0.3, 0.6, 1.1, 1.7, 3.5, 10.6, 31.2, 93.8]
+        ],
+        "center_freqs": [125, 250, 500, 1000, 2000, 4000, 8000, 16000, 32000],
+    }
+
+    def __init__(
+        self,
+        min_temperature: float = 10.0,
+        max_temperature: float = 20.0,
+        min_humidity: float = 30.0,
+        max_humidity: float = 90.0,
+        min_distance: float = 10.0,
+        max_distance: float = 100.0,
+        p=0.5,
+    ):
+        """
+        :param min_temperature: Minimum temperature in celcius
+        :param max_temperature: Maximum temperature in celsius
+        :param min_humidity: Minimum humidity in percent %
+        :param max_humidity: Maximum humidity in percent %
+        :param min_distance: Minimum microphone-source distance in meters.
+        :param max_distance: Maximum microphone-source distance in meters.
+        :param n_fft: The number of bins in the estimated filterbank
+        :param p: The probability of applying this transform
+        """
+        assert float(min_temperature) in [
+            10.0,
+            20.0,
+        ], "Sorry, the only supported temperatures are either 10 or 20 degrees celcius"
+        assert float(max_temperature) in [
+            10.0,
+            20.0,
+        ], "Sorry, the only supported temperatures are either 10 or 20 degrees celcius"
+        assert min_temperature <= max_temperature
+        assert 30 <= min_humidity <= max_humidity <= 90
+
+        super().__init__(p)
+
+        self.min_temperature = min_temperature
+        self.max_temperature = max_temperature
+
+        self.min_humidity = min_humidity
+        self.max_humidity = max_humidity
+
+        self.min_distance = min_distance
+        self.max_distance = max_distance
+
+    def randomize_parameters(self, samples, sample_rate):
+        super().randomize_parameters(samples, sample_rate)
+        self.parameters["temperature"] = 10 * np.random.randint(
+            int(self.min_temperature) // 10, int(self.max_temperature) // 10 + 1
+        )
+        self.parameters["humidity"] = np.random.randint(
+            self.min_humidity, self.max_humidity + 1
+        )
+        self.parameters["distance"] = np.random.uniform(
+            self.min_distance, self.max_distance
+        )
+
+    def apply(self, samples, sample_rate):
+        assert samples.dtype == np.float32
+
+        humidity = self.parameters["humidity"]
+        distance = self.parameters["distance"]
+
+        # Choose correct absorption coefficients
+        key = str(int(self.parameters["temperature"])) + "C"
+        bounds = [30, 50, 70, 90]
```

Comment:
So technically, we can choose only some values for the humidity and distance according to what is present in the table. I think it should be specified that we use only coefficients in `air_absorption_table`  in the docstring of this transform so that the user knows about it.

Reply:
The user can choose any value between 30 and 90 and the transform translates it to the relevant key. I will add a bit more about the boundaries though.

Reply:
I mean that the user should know that the transform translates it to the relevant key. :) If I were to use this transform, I would like to know if a transform maps my input to something else.

Reply:
I already specify I am using a look up table by pyroomacoustics.